disperser/cmd/apiserver/flags/flags.go

```diff
@@ -94,6 +94,13 @@ var (
 		EnvVar:   common.PrefixEnvVar(envVarPrefix, "RATE_BUCKET_STORE_SIZE"),
 		Required: false,
 	}
+	MaxBlobSize = cli.IntFlag{
```

Comment:
It looks no issue if different disperser replicas are using different MaxBlobSize.
Should we add validation to the Node to check if the blob size is under a limit?

Reply:
I don't think there is issue. But we want it, it has to be different deployment, which the current devops repo has yet offered that. 

I don't think we need to artificially constraint on blob size on the DA node size now, as far as the disperser isn't decentralized, but yes, once we got to the step. We need to have some rate limit or blob constraint. For now, I think we are good.

Empirically, there is no impact on validation speed with respect to the blob size.